### ISSUE TITLE
Migrate to Wagmi v2

### ISSUE THREAD

**Issue by Pabl0cks** [BuidlGuidl/abi.ninja#115]
We migrated SE-2 to Wagmi v2, but it is still pending on Abi Ninja. It will be a pre-requisite to develop interesting features in the future like https://github.com/BuidlGuidl/abi.ninja/issues/113, and always nice to have it synced with SE-2.

**Comment by technophile-04** [comment]
Fixed at #121
